### PR TITLE
Cgt 3250

### PR DESCRIPTION
**Pull request by janeodoi**

### CODE REVIEW

File: test/controllers/CalculationControllerTests/PropertyLivedInActionSpec.scala [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+    when(mockCalcConnector.saveFormData[PropertyLivedInModel](ArgumentMatchers.any(), ArgumentMatchers.any())(ArgumentMatchers.any(), ArgumentMatchers.any()))
+      .thenReturn(Future.successful(mock[CacheMap]))
+
+    new PropertyLivedInController {
+      override val calcConnector: CalculatorConnector = mockCalcConnector
+          }
+  }
+
+  "Calling .propertyLivedIn from the resident DeductionsController" when {
```
Comment (euptonqa) [inline test/controllers/CalculationControllerTests/PropertyLivedInActionSpec.scala]: Needs to be .propertyLivedIn from the PropertyLivedInController

File: test/views/PropertyLivedInViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
+import views.html.calculation.propertyLivedIn
+import play.api.i18n.Messages.Implicits._
+import play.api.Play.current
+import play.api.data.Form
+
+class PropertyLivedInViewSpec extends UnitSpec with WithFakeApplication with MockitoSugar with FakeRequestHelper with AssertHelpers {
+
+
+  "Bought for less view" when {
```
Comment (euptonqa) [inline test/views/PropertyLivedInViewSpec.scala]: Needs to be 'PropertyLivedIn view'.

File: test/controllers/CalculationControllerTests/PropertyLivedInActionSpec.scala (hunk 1 above)
Comment (janeodoi) [inline test/controllers/CalculationControllerTests/PropertyLivedInActionSpec.scala]: Updated with corrections

File: test/views/PropertyLivedInViewSpec.scala (hunk 2 above)
Comment (janeodoi) [inline test/views/PropertyLivedInViewSpec.scala]: Updated with correcions

Comment (janeodoi) [inline test/views/PropertyLivedInViewSpec.scala]: Ran Test and pushed changes to master

File: test/controllers/CalculationControllerTests/PropertyLivedInActionSpec.scala (hunk 1 above)
Comment (janeodoi) [inline test/controllers/CalculationControllerTests/PropertyLivedInActionSpec.scala]: Ran Test and pushed changes to master

**Pull request merged**